package.json

```diff
@@ -40,7 +40,7 @@
     "semver": "5.4.1",
     "triton-metrics": "1.0.3",
     "triton-netconfig": "1.5.0",
-    "ufds": "1.7.0",
+    "ufds": "git+https://github.com/TritonDataCenter/node-ufds.git#62f646a3d04f3703310564d8920857731d291058",
```

Comment:
Note for future us: Once the node-ufds change goes up (which one is that again?) this will need to be updated as well.